modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/ClusterManager.java

```diff
@@ -243,26 +241,6 @@ private void initClusterMode() throws AndesException {
             log.debug("Stored node ID : " + this.nodeId +  ". Stored node data(Hazelcast local "
                       + "member host address) : " + localMemberHostAddress);
         }
-
-        //add node information to durable store
-        andesContextStore.storeNodeDetails(nodeId, localMemberHostAddress);
-
-        /**
-         * If nodeList size is one, this is the first node joining to cluster. Here we check if
-         * there has been any nodes that lived before and somehow suddenly got killed. If there are
-         * such nodes clear the state of them and copy back node queue messages of them back to
-         * global queue. We need to clear up current node's state as well as there might have been a
-         * node with same id and it was killed
-         */
-        clearAllPersistedStatesOfDisappearedNode(nodeId);
-
-        List<String> storedNodes = new ArrayList<>(andesContextStore.getAllStoredNodeData().keySet());
-        List<String> availableNodeIds = clusterAgent.getAllNodeIdentifiers();
-        for (String storedNodeId : storedNodes) {
-            if (!availableNodeIds.contains(storedNodeId)) {
-                clearAllPersistedStatesOfDisappearedNode(storedNodeId);
-            }
-        }
```

Comment:
There are actually no persisted nodes in the database. What this did was adding its node id to the database right before clearing and removing it right after. So by the time this runs, only 1 node id is persisted in the database and it is this one itself. Moved the part that will close all subscripitions for the current node to the initialization of the subscription manager.